CHANGELOG.md

```diff
@@ -1,5 +1,64 @@
 # Changelog
 
+## 0.10.0
+
+API version - 0.1.18
+
+DB Schema version - 0.0.15
+
+This release contains a database schema update, but no data migration only new tables added.
```

Comment:
Nit: The second clause in this sentence would read a little better with a conjunction in it, something like "but no data migration as only new tables were added.